test/graphgym/test_graphgym.py

```diff
@@ -110,3 +110,77 @@ def test_run_single_graphgym(auto_resume, skip_train_eval, use_trivial_metric):
     agg_runs(cfg.out_dir, cfg.metric_best)
 
     shutil.rmtree(cfg.out_dir)
+
+
+@withPackage('yacs')
+@withPackage('pytorch_lightning')
+@pytest.mark.parametrize('auto_resume', [True, False])
+@pytest.mark.parametrize('skip_train_eval', [True, False])
+@pytest.mark.parametrize('use_trivial_metric', [True, False])
+def test_graphgym_module(auto_resume, skip_train_eval, use_trivial_metric):
+    import pytorch_lightning as pl
```

Comment:
consider moving this out with conditional import as this may do import with each test call (and this has quite large parametrizations)